cnf-certification-test/lifecycle/suite.go

```diff
@@ -36,19 +37,103 @@ var _ = ginkgo.Describe(common.LifecycleTestKey, func() {
 		provider.BuildTestEnvironment()
 		env = provider.GetTestEnvironment()
 	})
+	testContainersPreStop(&env)
+	testContainersImagePolicy(&env)
+	testContainersReadinessProbe(&env)
+	testContainersLivenessProbe(&env)
+	testPodsOwnerReference(&env)
+})
+
+func testContainersPreStop(env *provider.TestEnvironment) {
 	testID := identifiers.XformToGinkgoItIdentifier(identifiers.TestShudtownIdentifier)
 	ginkgo.It(testID, ginkgo.Label(testID), func() {
-		gomega.Expect(true).To(gomega.Equal(true))
 		badcontainers := []string{}
 		for _, cut := range env.Containers {
-			fmt.Println("container ", cut.Data.Name)
-			logrus.Debugln("check container ", cut.Data.Name) // maybe use different platform ?
+			logrus.Debugln("check container ", cut.Namespace, " ", cut.Podname, " ", cut.Data.Name, " pre stop lifecycle ")
```

Comment:
Shouldn't we also store this log in the claim file instead of logrus? I think that it would provide more detailed information

Reply:
What I did like about logrus is the ability to see logs when running a particular test in the console.
I will look into overriding logrus calls so that it will print to claim files or to the claim file. Jira ticket will be created to track this, if not successfull, then I'll switch all the logrus here into claim file

Reply:
I believe ClaimFilePrintf already prints both on console (same as the ginkgo output) and in the claim file

Reply:
I did not know that, in that case, I'll switch to tnf claim file printing

Reply:
```
**bad containers [test-6bfb745768-cvdt8 test-6bfb745768-lgjj9]**
------------------------------
• [FAILED] [0.332 seconds]
lifecycle
/Users/shamadi/redhat/go/cnf-certification-test/cnf-certification-test/lifecycle/suite.go:34
  [It] lifecycle-pod-owner-type [lifecycle-pod-owner-type]
  /Users/shamadi/redhat/go/cnf-certification-test/cnf-certification-test/lifecycle/suite.go:123

  Begin Captured GinkgoWriter Output >>
    STEP: Testing owners of CNF pod, should be replicas Set 02/13/22 16:24:49.684
    **bad containers [test-6bfb745768-cvdt8 test-6bfb745768-lgjj9]**
  << End Captured GinkgoWriter Output
  ```
  
  code between ** is being logged by ginkgo, It does actually log to stdout, it just does not have the notion of logging level in it.  We don't know if that's an error/trace/debug message. I'm not sure which way to go here.
  Either we can create our own abstraction and add level notion to "tnf.claimPrint" then inside it filter based on level, and print using logrus. 
  
  For now, I did send all the failed pods to Ginkgo, until we decide what to do :) We can even explore using zap or any other logging library if needed.